### ISSUE TITLE
fix repo description

### ISSUE THREAD

**Issue by ligthyear** [maidsafe/safe_launcher#121]
It says this was a "rust crate", while it is an electron/nodejs app now (is there an explanation of the way for this somewhere?). That is confusing me every time I open the page (making me believe I am on the wrong repo). Maybe we should update this?

![screen shot 2016-03-17 at 14 57 17](https://cloud.githubusercontent.com/assets/40496/13848118/b7f97474-ec50-11e5-8696-d3a6f9846910.png)

**Comment by Fraser999** [comment]
Hi there,

I've fixed the description - thanks for the heads-up.

I'm not sure exactly what you mean by "is there an explanation of the way for this somewhere?".

I'll close this issue now, but if you need more info on the above point, please feel free to re-open this or create a new issue.

Thanks again,
Fraser.

**Comment by ligthyear** [comment]
Thanks, @Fraser999 !

Sorry, that sentence wasn't really clear. Let me rephrase: I understand this used to be rust code, correct? But it is now a NodeJS/Electron app. And I was just wondering what the considerations for that decisions were and whether that had been documented somewhere. Just curious.

**Comment by krishnaIndia** [comment]
Initially when the launcher was developed in RUST the APIs were exposed through IPC via sockets, for which Rust was a best fit language to develop and we thought of building the UI on top of the launcher Rust library. But it was proving to be difficult when we had to cater for the web development because of varied restrictions imposed by the browsers.

Thus we had to revisit the initial plan and then we had a simpler work around using the proxy and exposing API as REST interface. For catering this, we wanted a platform/language that can provide easy integration with the common web stack (REST, Comet, Web Sockets, etc), making it easy for scaling in the future. Thus nodejs was opted for catering the needs and it also has lots of tested third party libraries which makes the integration easier. Still we use the ffi interface from the [safe_ffi](https://github.com/maidsafe/safe_ffi) crate for invoking the network operations.
